src/components/inline-inputs/inline-inputs.pw.tsx

```diff
@@ -0,0 +1,247 @@
+      mount,
+      page,
+    }) => {
+      await mount(<InlineInputComponent gutter={size} />);
+
+      await expect(inlineInputContainer(page)).toHaveCSS(
+        "margin-left",
+        `${margin}px`
+      );
+
+      await expect(inlineInputContainer(page)).toBeVisible();
+    });
+  });
+
+  testData.forEach((label) => {
+    test(`should check label as ${label}`, async ({ mount, page }) => {
+      await mount(<InlineInputComponent label={label} />);
+
+      await expect(inlineLabel(page)).toHaveText(label);
+    });
+  });
+
+  testData.forEach((classname) => {
+    test(`should check classname as ${classname}`, async ({ mount, page }) => {
+      await mount(<InlineInputComponent className={classname} />);
+
+      await containsClass(inlineInputsPreview(page), classname);
+    });
+  });
+
+  [30, 60, 50, 90].forEach((inputWidth) => {
+    test(`should check inputWidth as ${inputWidth}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<InlineInputComponent inputWidth={inputWidth} />);
+
+      await expect(inlineInputContainer(page)).toHaveCSS(
+        "flex",
+        `0 0 ${inputWidth}%`
+      );
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children}`, async ({ mount, page }) => {
+      await mount(
+        <InlineInputs label="Inline Input">
+          <Textbox>{children}</Textbox>
+        </InlineInputs>
+      );
+
+      await expect(inlineChildren(page)).toHaveText(children);
+
+      await expect(inlineChildren(page)).toBeVisible();
+    });
+  });
+
+  [45, 25, 35, 15].forEach((labelwidth) => {
+    test(`should check labelWidth as ${labelwidth}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<InlineInputComponent labelWidth={labelwidth} />);
+      await expect(inlinelabelWidth(page)).toHaveCSS(
+        "flex",
+        `0 0 ${labelwidth}%`
+      );
+    });
+  });
+
+  testData.forEach((htmlFor) => {
+    test(`should check htmlFor as ${htmlFor}`, async ({ mount, page }) => {
+      await mount(<InlineInputComponent htmlFor={htmlFor} />);
+      await expect(inlineLabel(page)).toHaveAttribute("for", htmlFor);
+    });
+  });
+});
+
+test.describe("Accessibility tests for InlineInputs component", () => {
+  test("should pass accessibility tests for InlineInputs Default example", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<Default />);
+
+    await checkAccessibility(page);
+  });
+
+  test("should pass accessibility tests for InlineInputs WithAdaptiveLabelBreakpoint example", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<WithAdaptiveLabelBreakpoint />);
+
+    await checkAccessibility(page);
+  });
+
+  test("should pass accessibility tests for InlineInputs Required example", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<Required />);
+
+    await checkAccessibility(page);
+  });
+});
+
+test.describe("rounded corners", () => {
+  ([
```

Comment:
comment (non-blocking): If you go with my suggestion above this and the test below could be replaced with `gutters.forEach`